Automattic/wp-calypso
# Refactor: move common import and importer modules to the blocks folder

#### Changes proposed in this Pull Request

* Refactor: move common `import` and `importer` modules from the signup framework to the `blocks`.
* Now, the shared modules live in the `blocks` folder, and the `signup` and `stepper` frameworks consume it from there.

#### Testing instructions

* e2e tests
* Just go to the `/start/importer/capture?siteSlug={SLUG}` and `/setup/import?siteSlug={SLUG}` and pass thought few flows and check if everything visually looks OK

Related to #62238
